LaunchAtLogin/copy-helper-swiftpm.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+package_resources_path="$BUILT_PRODUCTS_DIR/LaunchAtLogin_LaunchAtLogin.bundle/Contents/Resources"
+
+helper_name="LaunchAtLoginHelper"
+helper_path="$package_resources_path/$helper_name.zip"
+
+contents_path="$BUILT_PRODUCTS_DIR/$CONTENTS_FOLDER_PATH"
+login_items="$contents_path/Library/LoginItems"
+login_helper_path="$login_items/$helper_name.app"
+
+rm -rf "$login_helper_path"
+mkdir -p "$login_items"
+unzip "$helper_path" -d "$login_items/"
```

Comment:
Can you do a SHA265 checksum check on the ZIP before unzipping? Just to help make sure it's what the user expect. We can hard-code the SHA in the script here.